metadata_backend/api/handlers/object.py

```diff
@@ -213,13 +214,13 @@ async def delete_object(self, req: Request) -> Response:
         :raises: HTTPUnprocessableEntity if object does not belong to current user
         :returns: HTTPNoContent response
         """
+        schema_type = req.match_info["schema"]
+        accession_id = req.match_info["accessionId"]
+        LOG.info(f"Deleting object {schema_type} {accession_id}")
```

Comment:
line 264 mentions if we deleted or not this object, this line looks like a `.debug` log, or is there any reason we make it `.info` ?